### PR TITLE
Feat: Option to specify DU.zip download location

### LINKED ISSUE #18: Can't download DU.zip from behind a corporate firewall
I can't install this from behind our corporate firewall because all .zip downloads from external sources are blocked. 

Could the location of DU.zip be a configuration option?

Cypress do something similar, they have a CYPRESS_INSTALL_BINARY environment to specify the location - https://github.com/cypress-io/cypress/blob/3c286177cc57e699062162d9db9ba12a562e2ccf/cli/lib/tasks/install.js#L172.

### PR DESCRIPTION
**Pull request by markmclauchlan**
Add the ability to specify the download location of the DU.zip binary. 

This is useful for people working behind a corporate firewall where you can only download these things from internal repositories.

Resolves #18

### CODE REVIEW

File: get-sysinternals-du.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ if (process.platform !== 'win32') {
   process.exit(0)
 }
 
-https.get('https://download.sysinternals.com/files/DU.zip', function (res) {
+const duZipLocation = process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION ?? 'https://download.sysinternals.com/files/DU.zip';
```
Comment (simoneb) [inline get-sysinternals-du.js:10]: as that operator is only available in node 14+

```suggestion
const duZipLocation = process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION || 'https://download.sysinternals.com/files/DU.zip';
```

**Comment by simoneb** [comment]
@markmclauchlan please fix the build

**Comment by simoneb** [comment]
build is still broken

**Comment by markmclauchlan** [comment]
Updated again

**Review by simoneb — APPROVED** [review]

**Pull request merged**